collect_app/src/main/res/layout/form_download_error.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_margin="@dimen/margin_small"
+    app:cardElevation="2dp"
```

Comment:
Generally we've using `4dp` elevation. Do you think we should pull out a constant, or does it not really need to be consistent? I'll need to reread the Material Theming stuff.

Reply:
The default value was 1dp and I found it not enough. 2dp looks better but I can try 4 too. Where is that specified?

Reply:
I don't think we have it specified anywhere, it just seems we've been using 4dp a lot of places. I can play with it when we review error messages and maybe tweak text sizes.